resources/translations/cs_CZ/support/support_structure.md

```diff
@@ -0,0 +1,41 @@
+Struktura podp�ry
```

Comment:
This file contains lots of UTF-8 errors. Invalid continuation bytes, etc. Something went wrong in how you encoded this.